htdocs/js/MathQuill/mqeditor.js

```diff
@@ -468,17 +468,17 @@
 
 		if (latexEntryMode) return;
 
-		// Trigger an answer preview when the enter key is pressed in an answer box.
+		// Trigger a button press when the enter key is pressed in an answer box.
 		answerQuill.keydownHandler = (e) => {
 			if (e.key == 'Enter') {
 				// Ensure that the toolbar and any open tooltips are removed.
 				answerQuill.toolbar?.tooltips.forEach((tooltip) => tooltip.dispose());
 				answerQuill.toolbar?.remove();
 				delete answerQuill.toolbar;
 
-				// For ww2 homework
-				document.getElementById('previewAnswers_id')?.click();
-				// For gateway quizzes
+				// For ww2 homework, depends on $pg{options}{enterKey}
+				document.getElementById('enter_key_submit')?.click();
```

Comment:
Perhaps this should be changed to:
```suggestion
				const enterKeySubmit = document.getElementById('enter_key_submit');
				if (enterKeySubmit) enterKeySubmit.click();
				else document.getElementById('previewAnswers_id')?.click();
```
That way the standalone renderer can continue to function without any changes.  This will still work with the webwork2 changes in https://github.com/openwebwork/webwork2/pull/2119 (and without those changes).